src/components/Inputs/InputFile.js

```diff
@@ -0,0 +1,17 @@
+import { useRef } from 'react';
+export const InputFile = props => {
+    const { onChange } = props;
+    const fileRef = useRef();
+    const changeHandler = (e) => {
+      onChange(e.target.files[0]);
+    }
+  
+    return (
+      <div className="form-group">
+        <input 
+          type="file" 
+          onChange={changeHandler}
+          ref={props.fileRef} />

```

Comment:
Remove props. Add fileRef to deconstructor

Reply:
but in this file I save const fileRef = useRef() and import useRef. 
Should I add fileRef to deconstructor and delete userRef?